code/controllers/subsystem/blackbox.dm

```diff
@@ -282,7 +282,7 @@ Versioning
 	var/last_words = sanitizeSQL(L.last_words)
 	var/suicide = sanitizeSQL(L.suiciding)
 	var/map = sanitizeSQL(SSmapping.config.map_name)
-	if(!suicide && !first_death.len)
+	if(!(text2num(suicide)) && !first_death.len)

```

Comment:
maybe use `L.suiciding` directly?

Reply:
I like the sanitize for clearing out invalid values.

Reply:
Actually it barely matters.